website/content/docs/k8s/k8s-cli.mdx

```diff
@@ -2,73 +2,76 @@
+| <nobr>`-set-string`</nobr>    | String value that enables you to set a customizable string value. This flag is comparable to the `helm install --set-string` <br/> flag. You can use the `-set-string` flag multiple times to specify multiple strings. <br/> Consul Helm chart values are supported.                                                                                                                             | none                                    |
+| <nobr>`-timeout`</nobr>       | Specifies how long to wait for the installation process to complete before timing out. The value is specified with an integer and string value indicating a unit of time. <br/> The following units are supported: <br/> `ms` (milliseconds)<br/>`s` (seconds)<br/>`m` (minutes) <br/>In the following example, installation will timeout after one minute:<br/> `consul-k8s install -timeout 1m` | `10m`                                   |
+| <nobr>`-wait`</nobr>          | Boolean value that determines if Consul should wait for resources in the installation to be ready before exiting the command.                                                                                                                                                                                                                                                                     | `true`                                  |
+| <nobr>`-verbose`, `-v`</nobr> | Boolean value that specifies whether to output verbose logs from the install command with the status of resources being installed.                                                                                                                                                                                                                                                                | `false`                                 |
+| <nobr>`-help`, `-h`</nobr>    | Prints usage information for this option.                                                                                                                                                                                                                                                                                                                                                         | none                                    |
 
 See [Global Options](#global-options) for additional commands that you can use when installing Consul on Kubernetes.
 
 #### Example Commands
 
-The following example command installs Consul according in the `myNS` namespace according to the `secure` preset.
+The following example command installs Consul in the `myNS` namespace according to the `secure` preset.
 
 ```shell-session
```

Comment:
there's an extra space in before the `$` for the commands, can you remove them?

<img width="951" alt="image" src="https://user-images.githubusercontent.com/7556325/183508654-9faa0da6-d0a9-4f11-938a-b0c8acdfd22d.png">

Reply:
Done!